src/plugins/intel_cpu/src/nodes/random_uniform.cpp

```diff
@@ -0,0 +1,536 @@
+            if (p.work_amount == 0lu) {
+                return;
+            }
+            auto n = n_state + p.n_shift;
+            auto out_cur = out_u8 + p.dst_shift;
+            auto work_rest = static_cast<int64_t>(p.work_amount);
+            uint32_t res[4];
+
+#define EXEC_CASE(P)                                                                                    \
+            case element::P: {                                                                          \
+                auto out_t = reinterpret_cast<element_type_traits<element::P>::value_type *>(out_cur);  \
+                for (; work_rest > 0l; work_rest -= p.step, out_t += p.step) {                          \
+                    runPhilox(m_global_seed, counter, n, res);                                          \
+                    auto el_to_copy = std::min(p.step, static_cast<uint64_t>(work_rest));               \
+                    convertToOutputType(res, m_min_val.P, m_range_val.P, out_t, el_to_copy);            \
+                    if (++n == 0) {                                                                     \
+                        counter++;                                                                      \
+                    }                                                                                   \
+                }                                                                                       \
+            } break;
+
+            switch (m_output_prc) {
+                EXEC_CASE(f32)
+                EXEC_CASE(f16)
+                EXEC_CASE(bf16)
+                EXEC_CASE(i32)
+                EXEC_CASE(i64)
+                default: THROW_CPU_NODE_ERR("Unsupported type of RandomUniform: ", m_output_prc.to_string());
+            }
+
+#undef EXEC_CASE
+        };
+
+        parallel_nt(m_threads_num, threadBody);
+    }
+
+    // Calculate counter values for next RandomUniform run.
+    n_state += m_skip_count;
+    if (n_state < m_skip_count) {
+        counter_state++;
+    }
+
+    return { n_state, counter_state };
+}
+
+////////////// STL algo ///////////////
+void RandomUniform::computeStl(void* out, size_t work_amount) {
+    switch (m_output_prc) {
+        case element::f32: {
+            generateData<float, std::uniform_real_distribution<float>>(
+                    std::uniform_real_distribution<float>{m_min_val.f32, m_max_val.f32}, out, work_amount);
+        } break;
+        case element::i32: {
+            generateData<int32_t, std::uniform_int_distribution<int32_t>>(
+                    std::uniform_int_distribution<int32_t>{m_min_val.i32, m_max_val.i32}, out, work_amount);
+        } break;
+        case element::i64: {
+            generateData<int64_t, std::uniform_int_distribution<int64_t>>(
+                    std::uniform_int_distribution<int64_t>{m_min_val.i64, m_max_val.i64}, out, work_amount);
+        } break;
+        case element::f64: {
```

Comment:
The plugin doesn't support f64 and there are no such plans. So lets remove f64 instances

Reply:
Done